inlang/source-code/sdk/src/v2/mocks/messageBundles.ts

```diff
@@ -0,0 +1,152 @@
+import type { MessageBundle } from "../types.js"
+
+export const examplePluralFormal: MessageBundle = {
+	id: "mock-bundle-human-id",
+	alias: {
+		default: "mock-bundle-alias",
+	},
+	messages: [
+		{
+			locale: "en",
+			declarations: [
+				{
+					type: "input",
+					name: "username",
+					value: {
+						type: "expression",
+						arg: {
+							type: "variable",
+							name: "username",
+						},
+					},
+				},
+			],
+			selectors: [
+				{
+					type: "expression",
+					arg: {
+						type: "variable",
+						name: "count: plural",
```

Comment:
I don't think this expression is what you meant, this literally reads as "a variable called `count: plural`"

I think you wanted to apply the `plural` function to the variable `count`. 

The way to express this is by adding a function annotation to the expression:

```ts
{
  type : "expression",
  arg: { type : "variable", name: "count"},
  annotation: { type: "function", name: "plural", options: [] }
}
```

Reply:
That was my fault 😅 I naively created a mock message (which @jldec copied) without respecting the correct structure.